src/aks/Jenkinsfile

```diff
@@ -2,15 +2,15 @@ node('master') {
     def repository = 'https://github.com/ArieShout/todo-app-java-on-azure.git'
     def branch = 'aks'
```

Comment:
As discussed in email, we can remove this configuration and update the `SCM` stage with `checkout scm`